quantum/quantum_keycodes.h

```diff
@@ -63,6 +63,7 @@ enum quantum_keycodes {
     QK_ONE_SHOT_LAYER_MAX   = 0x54FF,
     QK_ONE_SHOT_MOD         = 0x5500,
     QK_ONE_SHOT_MOD_MAX     = 0x55FF,
+    QK_ONE_SHOT_STATUS      = 0x5600,
```

Comment:
I'm not sure that thati s the best way to implement this, especially without stepping on the existing oneshot code, or using a range just for itself.
 
Thinking about it, it may actually be best to move this to quantum, and be done with it. 

@fauxpark thoughts?

Reply:
Yeah i agree, using a whole block for 3 keycodes i kinda inefficient.

Reply:
My only issue is that I don't like the idea of keycodes for it being in quantum *and* tmk_core, but I'm not really sure that I could do otherwise, really.